input/pages/app-launch.md

```diff
@@ -474,16 +474,15 @@ Alternatively, the following example shows one way for a client app to cause the
 
 #### Response
 
-The authorization decision is up to the EHR authorization server,
-which may request authorization from the end-user. The EHR authorization
+The authorization decision is up to the EHR authorization server which may request authorization from the end-user. The EHR authorization
 server will enforce access rules based on local policies and optionally direct
 end-user input.
 
 The EHR decides whether to grant or deny access.  This decision is
 communicated to the app when the EHR authorization server returns an
-authorization code (or, if denying access, an error response).  Authorization codes are short-lived, usually expiring
+authorization code or, if denying access, an error response.  Authorization codes are short-lived usually expiring
```

Comment:
```suggestion
authorization code or, if denying access, an error response.  Authorization codes are short-lived, usually expiring
```

Need a `,` for a descriptive clause.

Reply:
See above ,  there is no natural pause in the sentence but I am OK either way.